### PR TITLE
Updated domain name (while keeping old one)

### PR DESCRIPTION
**Pull request by CaptainChicky**
This one just keeps the old one and adds the new one lol
// @match        *://chatgpt.com/*
// @match        *://chat.openai.com/*
idk

### CODE REVIEW

**Comment by 4as** [comment]
Each change to DeMod needs to be accompanied by a version bump. In this case version should be updated from 4.1 to 4.2 before I can accept the pull request.

File: ChatGPT DeMod.user.js [hunk 1]
```diff
@@ -4,6 +4,7 @@
 // @version      4.1
```
Comment (pukkandan) [inline ChatGPT DeMod.user.js:4]:
```suggestion
// @version      4.2
```

**Comment by pukkandan** [comment]
FYI, @4as, you can commit such small changes and merge the PR straight from the GH UI. I've found it's typically easier/faster than back and forth with contributor

**Comment by 4as** [comment]
Can I get a confirmation that after applying the fix DeMod still works on mobile (Android and iOS)?

**Comment by CaptainChicky** [comment]
Yes, I tested on iOS, and it works. I don't have an android phone so I'm not sure about android (it did work when i changed my browser footprint, although I do not think these are interchangable)

**Comment by CaptainChicky** [comment]
@4as Yep, I've done some testing on ios (i don't have an android phone though so i didnt test that) and it works. I mean regardless, what i did was just add an extra domain to match in the greasyfork script so there shouldn't be any issue.

**Review by 4as — APPROVED** [review]

**Comment by 4as** [comment]
Okay, great to hear. Thank you for providing a fix.

**Pull request merged**